Bots/DotNet/Skills/CodeFirst/WaterfallSkillBot/Dialogs/Sso/SsoSkillDialog.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.Bot.Builder;
 using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Builder.Dialogs.Choices;
+using Microsoft.Bot.Connector.Authentication;
```

Comment:
All these file changes, are already being address in the PR https://github.com/southworks/BotFramework-FunctionalTests/pull/96/files#diff-6e34c20106649b13cd3e3b9a03c7cd76f0c032061227dc0bff038d4ecff51177R65 for the WaterfallSkill